central/config/service/service.go

```diff
@@ -125,3 +128,45 @@ func (s *serviceImpl) PutConfig(ctx context.Context, req *v1.PutConfigRequest) (
 	}
 	return req.GetConfig(), nil
 }
+
+// GetVulnerabilityDeferralConfig returns Central's vulnerability deferral configuration.
+func (s *serviceImpl) GetVulnerabilityDeferralConfig(ctx context.Context, _ *v1.Empty) (*v1.GetVulnerabilityDeferralConfigResponse, error) {
+	if !env.UnifiedCVEDeferral.BooleanSetting() {
+		return nil, errors.Errorf("Cannot fulfill request. Environment variable %s=false", env.UnifiedCVEDeferral.EnvVar())
+	}
+	config, err := s.datastore.GetConfig(ctx)
+	if err != nil {
+		return nil, err
+	}
+	return &v1.GetVulnerabilityDeferralConfigResponse{
+		Config: VulnerabilityDeferralConfigStorageToV1(config.GetPrivateConfig().GetVulnerabilityDeferralConfig()),
+	}, nil
+}
+
+// UpdateVulnerabilityDeferralConfig updates Central's vulnerability deferral configuration.
+func (s *serviceImpl) UpdateVulnerabilityDeferralConfig(ctx context.Context, req *v1.UpdateVulnerabilityDeferralConfigRequest) (*v1.UpdateVulnerabilityDeferralConfigResponse, error) {
+	if !env.UnifiedCVEDeferral.BooleanSetting() {
```

Comment:
Questions for my own understanding. Why is `ROX_VULN_MGMT_UNIFIED_CVE_DEFERRAL` declared in `pkg/env` instead of `pkg/features`? Should the UI team follow the same pattern or continue to put our feature flags under `pkg/features/list.go`?

If we should put new flags under their respective `pkg/env` file, how do those flags get reported in the endpoint that the UI hits? I noticed that some `RegisterBooleanSetting` values under `pkg/env` are visible in the feature flag API, but some aren't, and there wasn't an immediately apparent reason why.

Reply:
UI does not need to make any changes. Calling the feature flags API will still return this env var. I am taking this approach of env var to help turn off the feature in production if you need it. Feature flags are baked into the release binaries. Therefore, they cannot be changed once set to a specific value.